grommet/grommet-index
# Localization/adding translations for CN and JP

What does this PR do?

Contribute translation of Grommet Strings to CN and JP

Where should the reviewer start?

the translation files

What testing has been done on this PR?

N/A

How should this be manually tested?

switch between browser locales

Any background context you want to provide?

We need it for our product :)

What are the relevant issues?

No translation currently exist in Grommet for CH and JP

Screenshots (if appropriate)

Do the grommet docs need to be updated?

Yes, you can update you are supporting 2 new locales

Should this PR be mentioned in the release notes?

Yes

Is this change backwards compatible or is it a breaking change?

Yes - backwards compatible
